Sources/MountebankSwift/Models/Stub/Response/Inject.swift

```diff
@@ -18,3 +18,9 @@ public struct Inject: StubResponse, Codable, Equatable {
         try injection.encode(to: encoder)
     }
 }
+
+extension Inject: Recreatable {
+    var recreatable: String {
+        "Inject(\(injection.debugDescription))"
```

Comment:
should this not be `recreatable` instead of `debugDescription`

Reply:
Would yield same output, since `injection` is a string and the `recreatable` imlementation of string uses `debugDescription`. But this should indeed use that implementation instead of `debugDescription` directly. Thanks

Reply:
Fixed (+ 2 more)